sql/0001-createphonebook.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE IF NOT EXISTS tbl_phonebook (
+    ID SERIAL PRIMARY KEY,
```

Comment:
better use lowercase convention

Reply:
It was in lower case originally. I'm got already confused in these code conventions.